yb_sync_LDAP/yb_sync_ldap.py

```diff
@@ -1169,10 +1169,12 @@ def run(self):
             logging.info('Run complete.')
         except LDAPSyncException as ex:
             print(json.dumps({"error": "LDAP Sync exception: {}".format(str(ex))}))
+            sys.exit(2)
         except Exception as ex:
             print(json.dumps({"error": "Exception: {}".format(str(ex))}))
             traceback.print_exc()
             traceback.print_stack()
+            sys.exit(3)
```

Comment:
Would it be better to use a high exit code like 125 for unknown exceptions? That would leave the lower numbers available in case additional specific exception handlers are implemented in future.